internal/worker/cmd_meta.go

```diff
@@ -258,6 +264,24 @@ var CommandsMeta = map[string]CmdMeta{
 	CmdCMSMerge: {
 		CmdType: SingleShard,
 	},
+	CmdHset: {
+		CmdType: SingleShard,
+	},
+	CmdHget: {
+		CmdType: SingleShard,
+	},
+	CmdHsetnx: {
+		CmdType: SingleShard,
+	},
+	CmdHdel: {
+		CmdType: SingleShard,
+	},
+	CmdHmset: {
+		CmdType: SingleShard,
+	},
+	CmdHmget: {
+		CmdType: SingleShard,
+	},
```

Comment:
Please update here as per new vars

Reply:
can you elaborate more on this?

Reply:
When you rename the vars above, this will be needing an update here as well